index.html

```diff
@@ -14,11 +14,61 @@
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
+
 <body>
   <header>
+    <a href="./index.html" class="logo">
+      <h2 class="header-h2">Code Journal</h2>
+    </a>
+    <div class="header" data-view="entry-form">
+
+    </div>
   </header>
+
   <main>
+    <div class="main-header" data-view="entry-form">
```

Comment:
All of your content for this view should be in one element with the `data-view` attribute, not an individual one for each chunk of code.